pkg/kubelet/config/common_test.go

```diff
@@ -66,10 +66,10 @@ func TestDecodeSinglePod(t *testing.T) {
 		Status: v1.PodStatus{
 			PodIP: "1.2.3.4",
 			PodIPs: []v1.PodIP{
-				v1.PodIP{
```

Comment:
back to kubenet commit

Reply:
This is actually part of kubenet, not the api. the api covers everything types related (except kubelet related changes.)